contrib/services/systemd/dcrd.service

```diff
@@ -1,13 +1,53 @@
 [Unit]
 Description=Decred Full Node
+Requires=local-fs.target network-online.target network.target
+After=local-fs.target network-online.target network.target
 
 [Service]
 Type=simple
+
+# modify to your username
```

Comment:
This is intentionally not the user's account.  It is expected that the service will be run under a separate account and group which is best practice for privilege separation.